source/partials/autopilot/custom-scheduling.md

```diff
@@ -0,0 +1,21 @@
+---
+contenttype: [partial]
+categories: [update, test]
+cms: [--]
+product: [autopilot]
+integration: [--]
+tags: [--]
+reviewed: "2023-03-17"
+---
+
+Custom Frequency Scheduling is available to sites owned by Platinum and Diamond Workspaces. This functionality allows you to schedule Autopilot to run on a more specific cadence. For example, on a particular day of the week, or day of the month.
```

Comment:
Since this appears directly below the note about Custom Scheduling only being for Platinum and Diamond workspaces, this note feels a little repetitive. Maybe we remove the first sentence and condense it to say "Custom Frequency Scheduling allows you to schedule Autopilot..." ?

Reply:
Done!